lib/std/math/big/int.zig

```diff
@@ -2660,57 +2660,56 @@ pub const Managed = struct {
 
     /// r = a + scalar
     ///
-    /// r and a may be aliases. If r aliases a, then caller must call
-    /// `r.ensureAddScalarCapacity` prior to calling `add`.
-    /// scalar is a primitive integer type.
```

Comment:
Did you intend to delete this line or was it deleted accidentally?

Reply:
Intended. It was replaced with the sentence " r and a may be aliases". The other part is no longer a requirement of the caller.

Reply:
So `scalar` can now be something else than a primitive integer type?